### PR TITLE
Harish Gumnur Blog

### PR DESCRIPTION
**Pull request by harry7337**
Introduction to Deep Learning

### CODE REVIEW

File: _posts/2022-04-12-Intro-To-Deep-Learning.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+![Drawback](/blog/assets/img/intro-to-dl/drawback.png)
+
+Souce: edureka
+
+
+Traditional ML algorithms are not useful while working with high dimensional data, that is where we have a large number of inputs and outputs. For example, in the case of facial recognition, there are copious amounts of data that need to be dealt with. ML simply cannot provide a satisfactory result relative to the amount of data that is being provided to the program.
+
+
+Another challenge is to extract the features or parameters in our data that play a vital role in predicting the outcome. This very process is referred to as feature extraction. ML does not seem to be able to efficiently extract features from the data we provide it. Let us take the example of image recognition, the patterns present in the image cannot be extracted satisfactorily with ML, and it fails to efficiently recognize important features in an image, like a face, scenery,etc…
+Overcoming challenges with DL
```
Comment (nishant-nayak) [inline _posts/2022-04-12-Intro-To-Deep-Learning.md:79]: Was this meant to be a heading?

**Review by nishant-nayak — CHANGES_REQUESTED** [review]
Looks pretty good! If the one comment I provided was not meant to be a heading, it looks good for publishing.

**Review by nishant-nayak — APPROVED** [review]
Looks good to merge!

File: _posts/2022-04-12-Intro-To-Deep-Learning.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+### Computer games:
+#### Go
+The standard Go board has a 19×19 grid of lines, containing 361 points. Despite its relatively simple rules, Go is extremely complex. 
+Compared to chess, Go has both a larger board with more scope for play and longer games and, on average, many more alternatives to consider per move. 
+The number of legal board positions in Go has been calculated to be approximately 2.1×10170, which is vastly greater than the number of atoms in the observable universe, estimated to be of the order of 1080.
+
+DeepMind’s AlphaGo defeated Lee Sedol, one of the best Go players in the world!
+
+![GO](/blog/assets/img/intro-to-dl/go.jped)
```
Comment (nishant-nayak) [inline _posts/2022-04-12-Intro-To-Deep-Learning.md:185]: Please fix this image link, should be `go.jpeg`

**Review by nishant-nayak — CHANGES_REQUESTED** [review]
Didn't see the image change last time around, please fix this

**Review by harry7337 — COMMENTED** [review]
Resolved image link

**Review by nishant-nayak — APPROVED** [review]

**Pull request merged**